tests/nativeapp/test_event_sharing.py

```diff
@@ -487,11 +519,22 @@ def test_event_sharing_disabled_no_change_to_current_behavior(
         console=mock_console,
     )
 
-    assert mock_execute_query.mock_calls == mock_execute_query_expected
-    mock_console.warning.assert_not_called()
+    assert [
+        *mock_execute_query.mock_calls,
+        *mock_sql_facade_upgrade_application.mock_calls,
+        *mock_sql_facade_create_application.mock_calls,
+    ] == expected
+
+    mock_console.warning.assert_called_once_with(
```

Comment:
why is application creation a warning? or has it always been the case?

Reply:
I believe it's always been the case if we look [here](https://github.com/snowflakedb/snowflake-cli/pull/1870/files#diff-e9438874d872da0c1e6711d40f24dd6162ad6cb31684b46eb61688a8503237bbL336-R333), but we've always had the mock return None, which I don't think reflects reality, so I updated the tests for that